src/transformers/generation/tf_utils.py

```diff
@@ -1230,7 +1230,7 @@ def _prepare_input_ids_for_generation(
     ) -> tf.Tensor:
         if self.config.is_encoder_decoder and encoder_outputs is not None:
             # make dummy input_ids with value -100, as a sanity check ensuring that they won't be used for encoding
-            shape = encoder_outputs.last_hidden_state.size()[:-1]
+            shape = encoder_outputs.last_hidden_state.shape[:-1]
```

Comment:
The power of copy-paste: I added PT syntax in TF code in the last PR 🙃 Fortunately, the tests added in this PR caught it.